deps/undici/src/index-fetch.js

```diff
@@ -26,6 +26,10 @@ module.exports.createFastMessageEvent = createFastMessageEvent
 
 module.exports.EventSource = require('./lib/web/eventsource/eventsource').EventSource
 
+const api = require('./lib/api')
+const Dispatcher = require('./lib/dispatcher/dispatcher')
+Object.assign(Dispatcher.prototype, api)
```

Comment:
why is this needed?

Reply:
I opened a separate PR in undici https://github.com/nodejs/undici/pull/4064